shopinvader_customer_multi_user/tests/common.py

```diff
@@ -22,6 +24,13 @@ def setUpClass(cls):
         )
         cls.backend.multi_user_records_policy = "record_id"
 
+    @staticmethod
```

Comment:
I've done something similar here https://github.com/shopinvader/odoo-shopinvader/commit/b94138f8638ef277a2b95e4c2ceb18a17014ac29 (which should fix the same issue of this PR).

Maybe better to add this to the common utils mixin and call it `_get_random_ext_id`?